python/ray/util/sgd/torch/worker_group.py

```diff
@@ -20,6 +21,7 @@
 logger = logging.getLogger(__name__)
 
 
+@DeveloperAPI
```

Comment:
Same as above, we shouldn't need any annotations for this. This is purely internal.